src/components/containers/seo.js

```diff
@@ -92,6 +93,9 @@ const SEO = ({ description, meta, title, no_index, has_organization_schema, meta
             htmlAttributes={{
                 lang: formatted_lang,
             }}
+            bodyAttributes={{
+                dir: i18next.dir(i18next.language),
```

Comment:
Not sure how often we need to get the direction of the language but can we have a quick helper function of this?

Reply:
you are absolutely right @prince-deriv , in fact I have the function, didn't use it. will use it here as well, thanks :pray: